src/icrar/leap-accelerate/config.h.in

```diff
@@ -1,2 +1,11 @@
 #define PROJECT_SOURCE_DIR "@PROJECT_SOURCE_DIR@/"
-#define PROJECT_BINARY_DIR "@CMAKE_BINARY_DIR@/bin/"
\ No newline at end of file
+#define PROJECT_BINARY_DIR "@CMAKE_BINARY_DIR@/bin/"
+
+/// The major version of this package
+#define LEAP_ACCELERATE_VERSION_MAJOR @LEAP-Accelerate_VERSION_MAJOR@
```

Comment:
I would have called this something like PROJECT_VERSION_MAJOR for easier code reuse but this is fine

Reply:
Which one do you refer to? I used `LEAP_ACCELERATE_` to prefix the macro name instead of a more generic `PROJECT_` to make it more meaningful -- each library usually exposes its own version numbers so namespacing these macros makes it easier to distinguish which one is being used.

The cmake variable is named like that in the top-level `CMakeLists.txt` file and I wanted to keep changes to a minimum.

Anyhow, thanks for the review! I'll merge to the master branch now.